src/modules/Action.js

```diff
@@ -24,7 +25,9 @@ type ActionType =
   // | 'OPEN_VIEWXPUB_WALLET_MODAL'
   // | 'CLOSE_VIEWXPUB_WALLET_MODAL'
   // | 'UNLOCK_WALLET_SEED'
-  | 'SWAP_FROM_TO_CRYPTO_WALLETS'
+  // | 'SELECT_FROM_WALLET_CRYPTO_EXCHANGE'
+  // | 'SELECT_TO_WALLET_CRYPTO_EXCHANGE'
+  // | 'SWAP_FROM_TO_CRYPTO_WALLETS'
```

Comment:
Should we leave these commented lines here? Or can they be removed?

Reply:
When Kevin initially did this work, he had comments for all the properly-typed actions. I don't think this is necessary, but I didn't want to rock the boat, so I left them in.

Reply:
I went ahead and removed them, per your comment.